packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2411,6 +2457,8 @@ class Navigator extends StatefulWidget {
   /// the initial route.
   ///
   /// If there is no [Navigator] in scope, returns false.
+  ///
+  /// Does not consider [PopScope].
```

Comment:
For my own understanding: canPop also never considered WillPopScope, right?

Reply:
That's right.  I think I added this because I was figuring things out myself and thought it would have been useful to mention it in the docs.